### PR TITLE
cleanup logging to be able to see complete RTA flow

### PR DESCRIPTION
**Pull request by mbg033**

### CODE REVIEW

File: include/rta/fullsupernodelist.h [hunk 1]
```diff
@@ -126,6 +126,9 @@ class FullSupernodeList
     size_t refreshedItems() const;
 
 
+    static std::string printAuthSample(const std::vector<SupernodePtr> &authSample);
```
Comment (bitkis) [inline include/rta/fullsupernodelist.h]: why don't you use `operator<<` instead??

**Comment by mbg033** [comment]
@bitkis please check the changes

File: include/rta/fullsupernodelist.h (hunk 1 above)
Comment (mbg033) [reply include/rta/fullsupernodelist.h]: replaced with operator<<

File: src/requests/authorizertatxrequest.cpp [hunk 2]
```diff
@@ -425,43 +453,53 @@ Status handleRtaAuthResponseMulticast(const Router::vars_t& vars, const graft::I
...
-                MDEBUG("  rta signatures: ");
-                std::string buf;
-                buf += "\n";
-                for (const auto & rta_sign:  tx.rta_signatures) {
-                    buf += string("      address: ") + cryptonote::get_account_address_as_str(true, rta_sign.address) + "\n";
-                    buf += string("      signature: ") + epee::string_tools::pod_to_hex(rta_sign.signature) + "\n";
-                }
-                MDEBUG(buf);
-            }
+//            {
```
[the remark this answers is not in the document]
Comment (mbg033) [reply src/requests/authorizertatxrequest.cpp]: done

**Pull request merged**